tests/widgets/test_colour_text.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+
+import unittest
+from urwid import AttrSpec
+from unittest import mock
+
+from alot.settings import theme
+from alot.widgets.colour_text import parse_quotes
+
+
+DUMMY_THEME_STRING = """\
+[bufferlist]
+    line = '', '', '', '', '', ''
+    line_even = '', '', '', '', '', ''
+    line_focus = '', '', '', '', '', ''
+    line_odd = '', '', '', '', '', ''
+[envelope]
+    body = '', '', '', '', '', ''
+    header = '', '', '', '', '', ''
+    header_key = '', '', '', '', '', ''
+    header_value = '', '', '', '', '', ''
+[global]
+    body = '', '', '', '', '', ''
+    footer = '', '', '', '', '', ''
+    notify_error = '', '', '', '', '', ''
+    notify_normal = '', '', '', '', '', ''
+    prompt = '', '', '', '', '', ''
+    tag = '', '', '', '', '', ''
+    tag_focus = '', '', '', '', '', ''
+[help]
+    section = '', '', '', '', '', ''
+    text = '', '', '', '', '', ''
+    title = '', '', '', '', '', ''
+[taglist]
+    line_even = '', '', '', '', '', ''
+    line_focus = '', '', '', '', '', ''
+    line_odd = '', '', '', '', '', ''
+[namedqueries]
+    line_even = '', '', '', '', '', ''
+    line_focus = '', '', '', '', '', ''
+    line_odd = '', '', '', '', '', ''
+[search]
+    focus = '', '', '', '', '', ''
+    normal = '', '', '', '', '', ''
+    [[threadline]]
+        focus = '', '', '', '', '', ''
+        normal = '', '', '', '', '', ''
+[thread]
+    quote_level_2 = '', '', '', '', '#201', '#202'
+    quote_level_3 = '', '', '', '', '#301', '#302'
+    quote_level_4 = '', '', '', '', '#401', '#402'
+    quote_level_5 = '', '', '', '', '#501', '#502'
+    quote_level_7 = '', '', '', '', '#701', '#702'
+    arrow_bars = '', '', '', '', '', ''
+    arrow_heads = '', '', '', '', '', ''
+    attachment = '', '', '', '', '', ''
+    attachment_focus = '', '', '', '', '', ''
+    body = '', '', '', '', 'h102', 'h103'
+    header = '', '', '', '', '', ''
+    header_key = '', '', '', '', '', ''
+    header_value = '', '', '', '', '', ''
+    [[summary]]
+        even = '', '', '', '', '', ''
+        focus = '', '', '', '', '', ''
+        odd = '', '', '', '', '', ''
+"""
+
+DUMMY_THEME = theme.Theme(DUMMY_THEME_STRING.splitlines())
+
+SETTINGS = {'quote_symbol': '>',
+            'colourmode': 256
+            }
+
+SETTINGS_test_symbol_regex = {'quote_symbol': '([Aa]|<>.b|>)',
```

Comment:
As mentioned in another comment, this is a weird thing to test for. Maybe we don't need the code to be that generic.